dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -287,6 +287,11 @@ def get_airflow_extras():
     "trino",
 ]
 ALL_PROVIDER_YAML_FILES = Path(AIRFLOW_SOURCES_ROOT).glob("airflow/providers/**/provider.yaml")
+
+PROVIDER_DEPENDENCIES = json.loads(
+    (Path(AIRFLOW_SOURCES_ROOT) / "generated" / "provider_dependencies.json").read_text()
+)
```

Comment:
```suggestion
with Path(AIRFLOW_SOURCES_ROOT, "generated", "provider_dependencies.json").open() as f:
    PROVIDER_DEPENDENCIES = json.load(f)
```

Marginally better performance.